apps/total-typescript/src/middleware.ts

```diff
@@ -10,19 +8,12 @@ const PUBLIC_FILE = /\.(.*)$/
 
 // The allow-list of paths where this middleware executes (perf)
 export const config = {
-  matcher: ['/', '/workshops/:module/:section/:lesson*'],
+  matcher: ['/'],
 }
 
-export default withAuth(
```

Comment:
`withAuth` works fine, but our next-auth config loads data using prisma and that uses Jose under the hood and calls forbidden APIs in edge functions

solution would be to use planetscale driver and load users with SQL in next-auth config, which is probably good idea